WorkoutTest/Views/TimerView.swift

```diff
@@ -23,7 +23,7 @@ class TimerView: UIView {
     let timerLabel = UILabel()
     timerLabel.font = UIFont.systemFont(ofSize: 150)
     timerLabel.textColor = .cyan
-    timerLabel.text = "10"
+    timerLabel.text = "3"
```

Comment:
Remove setting a text from here at all. It is not needed to be here. All you need to do - is to set the correct label in the ExerciseViewController in `func setupTimerView()` 
`timerView.timerLabel.text = "\(secondRemaining)"`